### ISSUE TITLE
TV App Programm Icon

### ISSUE THREAD

**Issue by Mrpink200** [zipleen/tvheadend-iphone-client#241]
Hello. Many thanks for this great app.
I ve seen that programm icons provided by epg are showing in IOS app but not in the Apple TV 4 app. 
Could it be updated ?

+ One great improvement would be to be able to add an option to replace TV channel icon by programm icon in the channel menu when available. My xml epg programm add this and it would be really great

As an example : 
```
<programme start="20190115001500 +0100" stop="20190115012000 +0100" showview="" channel="C4.api.telerama.fr">
    <title>Les enfants de la télé</title>
    <desc lang="fr">Chaque dimanche, Laurent Ruquier fait vivre aux téléspectateurs et aux personnalités présentes sur le plateau, un moment convivial d'humeur et d'humour autour de l'actualité artistique, culturelle et télévisuelle, dans un savant mélange de nostalgie, d'actualité et de décryptages.</desc>
    <credits>
      <presenter>Laurent Ruquier</presenter>
      <guest>Sylvie Tellier</guest>
      <guest>Vaimalama Chaves</guest>
      <guest>Titoff</guest>
      <guest>Jean-Michel Ribes</guest>
    </credits>
    <category lang="fr">divertissement</category>
    <length units="minutes">65</length>
    **<icon src="http://television.telerama.fr/sites/tr_master/files/sheet_media/media/169_COL_845091.jpg" />**
    <video>
      <aspect>16:9</aspect>
    </video>
    <previously-shown />
    <rating system="CSA">
      <value>Tout public</value>
    </rating>
```

my epg source : http://www.xmltv.fr

Best Regards

**Comment by zipleen** [comment]
I see.. that's because the EPG does not exist in ATV, only "guide" which does not contain pictures (well it does but only if you go into).

You do have an interesting point, although I have no idea how to test it.. join http://chat.tvhclient.com if you want to join the conversation

**Comment by Mrpink200** [comment]
I can t access the chat at work sorry. it s strange as i get the full text epg working on my setup :s

**Comment by Mrpink200** [comment]
i could help on tomorrow as i will be on a different network

**Comment by zipleen** [comment]
yes of course, but what I'm saying is that there's no place to put the image. However you do have a good point, the image could replace the channel logo if the currently running program has one. 

However what if this makes finding the channel harder? Because in the program listing there's no place that has the channel name, only the show and time... I could add the name of the channel on the bottom right though, but I would need to test stuff out to see if it gets too crowded

**Comment by Mrpink200** [comment]
i would be happy to test, but my conf is not really standard as i use my apple tv on a video projector. Perhaps it could be first just an option as you did for channel icon backgroud color, no mess for people who does not want that.
